R/ff_prep_refactored.R

```diff
@@ -424,7 +368,7 @@ finalize_data <- function(dts, selected_files, addxy, adddate) {
   return(list(dts = dts, newcolnames = newcolnames))
 }
 
-sample_and_combine_data <- function(dts, fdts, sf_indices, sample_size, first, allindices, verbose) {
```

Comment:
We could consider leaving verbose in. I know it's not used in the current definition, however, if we keep it in every function we might as well give the option here. Another idea is to remove verbose from all the sub-function definitions and define it as a global variable when ff_prep is called?

Reply:
global variables in R is not recommended as a best practice, also verbose is able to change through ff_prep interface. maybe it would be good idea to have it in configuration file?

Reply:
Configuration file doesn't feel like the correct place for this imo as it might change from run to run as it's main purpose is bug fixing I think

Reply:
I only see the reference at the top but that also says "use camelcase for var names" which is sth we don't use either. I think those are just examples. I don't see tidyverse or the google R guide say anything about it. But yeah we can discuss it.